project/src/main/webapp/script.js

```diff
@@ -400,3 +407,29 @@ function registerUserByToken() {
   }
   fetch('/register?idToken=' + googleUser.getAuthResponse().id_token, {method: 'POST'});
 }
+
+/** Sends feedback that includes the place the user chose. */
+function sendUserChoiceAsFeedback() {
+  let indexOfPlaceUserChose = getCheckedValueByElementId('chosen-place-form',
+      'Please select the place that you chose.');
+  if (indexOfPlaceUserChose <= recommendedPlaces.length) {
+    storeUserFeedback(recommendedPlaces, recommendedPlaces[indexOfPlaceUserChose].placeId,
+        false /** user tried again */)
+  }
+}
+
+/** Stores user feedback in the database, by using the feedback servlet. */
+function storeUserFeedback(recommendedPlaces, chosenPlaceId, tryAgain) {
```

Comment:
nit: why "store"? Maybe "update" or "post" would be more meaningful here, as you don't actually store anything (the backend does, but it's not up to the JS code to know that).

Reply:
I used "update" and Tal said in the comments here: "I think the name of this function is a little confusing. It stores the feedback right? It doesn't update a feedback already existing. So maybe change to storeUserFeedback."
She has a point... I mean, the function is called in order to start the process of storing the data. 
I think maybe post is a good idea and clearer than the others? WDYT?

Reply:
done, used post :)